components/payments/settings/payment-webhooks.tsx

```diff
@@ -30,6 +33,12 @@ export const PaymentsWebhooks: React.FC<PaymentsWebhooksProps> = ({
   const { mutate: updateWebhook, isLoading: isUpdateLoading } = usePaymentsUpdateWebhook(accountId);
   const { mutate: createWebhook, isLoading: isCreateLoading } = usePaymentsCreateWebhook(accountId);
 
+  const [secretKey, setSecretKey] = React.useState<string>("");
+  React.useEffect(() => {
```

Comment:
why do you need this in state at all? can you just... define it once? it will get re-computed at every render and if don't' want that to happen just put it outside the component iself (like on line 15)

Reply:
Just a filler key for now